blob/rollup.config.js

```diff
@@ -0,0 +1,49 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import { uglify } from 'rollup-plugin-uglify';
+import builtins from 'rollup-plugin-node-builtins';
+import replace from 'rollup-plugin-replace';
+import globals from 'rollup-plugin-node-globals';
+
+export default [
+  {
+    external: ['ms-rest-js', 'crypto', 'fs', 'events', 'os'],
+    input: 'dist-esm/lib/index.js',
+    output: {
+      file: 'dist/index.js',
+      format: 'cjs',
+      sourcemap: true
+    },
+    plugins: [nodeResolve({ module: true }), uglify()]
+  },
+  {
+    external: ['ms-rest-js'],
+    input: 'dist-esm/lib/index.browser.js',
+    output: {
+      globals: {
+        'ms-rest-js': 'msRest'
+      },
+      file: 'dist/index.browser.js',
+      format: 'umd',
+      name: 'azBlob',
```

Comment:
`azblob` is what exposed, we'd better avoid breaking until an agreement by other Azure teams. (Any updates for this naming discussion with other Azure teams?)

Reply:
Fixed